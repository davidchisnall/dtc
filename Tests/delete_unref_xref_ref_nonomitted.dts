/dts-v1/;

/ {
	/omit-if-no-ref/ qux0: qux@0 {
		test-property = <&qux1>;
		test-property-ng = <&quux>;
	};
	/omit-if-no-ref/ qux1: qux@1 {
		test-property = <&qux0>;
	};

	quux: quux@2 { };
};
