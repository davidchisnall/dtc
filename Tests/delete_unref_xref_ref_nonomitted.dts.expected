/dts-v1/;

/  {

	quux@2 {

	};
};
